core/src/main/java/org/apache/accumulo/core/client/admin/ActiveScan.java

```diff
@@ -96,4 +97,11 @@ public abstract class ActiveScan {
    * @since 1.5.0
    */
   public abstract long getIdleTime();
+
+  /**
+   * Return the host where the compaction is running.
+   *
+   * @since 4.0.0
+   */
+  public abstract ServerId getHost();
```

Comment:
There sevrver id contains more info than just the hostname, so this getter may need a different name.

```suggestion
  public abstract ServerId getServeId();
```

Reply:
The reason why I chose that name was because that's what ActiveCompaction [uses](https://github.com/apache/accumulo/blob/5147077ff1535294f469a52732a3b0f3d985c236/core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java#L141) so I was trying to be consistent. Maybe that one should be deprecated and renamed as well.

Reply:
Hmm, not sure what is going on with ActiveCompactions because the SeverId type did not exist in 2.1.0.  So not sure how that method has a since 2.1.0 and returns ServerId.

Reply:
looks like this was done in https://github.com/apache/accumulo/pull/4963, I will change this based on the suggestion [here](https://github.com/apache/accumulo/pull/4963#discussion_r1792629478) as that makes more sense.